phone-scripts/02-install-packages.sh

```diff
@@ -29,14 +29,11 @@ infecho "THIS WILL FAIL, DON'T WORRY ITS PROBABLY OK"
 dnf -y remove kernel || rpm -e --noscripts kernel-core
 dnf -y install linux-firmware
 
-infecho "Installing kernel..."
-dnf -y --disablerepo="*" --enablerepo="copr:copr.fedorainfracloud.org:njha:mobile" install megi-kernel
-
 infecho "Installing recommended packages..."
-dnf -y install feedbackd phoc phosh squeekboard gnome-shell ModemManager rtl8723cs-firmware \
+dnf -y install megi-kernel feedbackd phoc phosh squeekboard gnome-shell ModemManager rtl8723cs-firmware \
     f2fs-tools chatty calls carbons purple-mm-sms pinephone-helpers evolution-data-server \
-    f32-backgrounds-gnome kgx epiphany gnome-contacts evolution cheese NetworkManager-wwan \
-    lightdm-mobile-greeter firefox
+    f33-backgrounds-gnome kgx epiphany gnome-contacts evolution NetworkManager-wwan \
+    lightdm-mobile-greeter firefox mautilus 
```

Comment:
Adding nautilus. It doesn't come with an adaptive UI, but it does "sort" of work, and it is useful in Convergence mode.